packages/web-components/src/components/page-header/page-header-breadcrumb.ts

```diff
@@ -33,20 +33,101 @@ class CDSPageHeaderBreadcrumb extends LitElement {
   @property({ attribute: 'within-grid', type: Boolean })
   withinGrid = false;
 
-  render() {
-    const { withinGrid } = this;
+  /**
+   * Set to `true` if page actions should be flush (no padding)
+   */
+  @property({ attribute: 'page-actions-flush', type: Boolean })
+  pageActionsFlush = false;
 
+  /**
+   * Set to `true` if content actions should be flush (no padding)
+   */
+  @property({ attribute: 'content-actions-flush', type: Boolean })
+  contentActionsFlush = false;
+
+  /**
+   * Set to `true` if there are content actions
+   */
+  @state()
+  private _hasContentActions = false;
+
+  /**
+   * Set to `true` if there are page actions
+   */
+  @state()
+  private _hasPageActions = false;
+
+  /**
+   * Handles `slotchange` event for content actions.
+   */
+  protected _handleContentActionsSlotChange({ target }: Event) {
+    this._hasContentActions = Boolean(
+      (target as HTMLSlotElement).assignedNodes().length
+    );
+    this.requestUpdate();
+  }
+
+  /**
+   * Handles `slotchange` event for page actions.
+   */
+  protected _handlePageActionsSlotChange({ target }: Event) {
+    this._hasPageActions = Boolean(
+      (target as HTMLSlotElement).assignedNodes().length
+    );
+    this.requestUpdate();
+  }
+
+  render() {
+    const {
+      border,
+      withinGrid,
+      pageActionsFlush,
+      contentActionsFlush,
+      _hasContentActions: hasContentActions,
+      _hasPageActions: hasPageActions,
+      _handleContentActionsSlotChange: handleContentActionsSlotChange,
+      _handlePageActionsSlotChange: handlePageActionsSlotChange,
+    } = this;
+    console.log(border);
     const gridClasses = classMap({
-      [`${prefix}--grid`]: !withinGrid,
+      [`${prefix}--css-grid`]: !withinGrid,
       [`${prefix}--subgrid ${prefix}--subgrid--wide`]: withinGrid,
     });
 
+    const breadcrumbBarClasses = classMap({
+      [`${prefix}--page-header__breadcrumb-bar`]: true,
+      [`${prefix}--page-header__breadcrumb-bar-border`]: border,
+      [`${prefix}--page-header__breadcrumb__actions-flush`]: pageActionsFlush,
+    });
+
+    const contentActionsClasses = classMap({
+      [`${prefix}--page-header__breadcrumb__content-actions`]:
+        !contentActionsFlush,
+    });
```

Comment:
```suggestion
    const contentActionsClasses = classMap({
      [`${prefix}--page-header__breadcrumb__content-actions`]:
        !contentActionsFlush,
    });
```

I'm thinking we can probably removed this? Since you are applying the styles via the `::slotted` selector?